### ISSUE TITLE
Add device path to `bcachefs show-super`

### ISSUE THREAD

**Issue by Managor** [koverstreet/bcachefs-tools#330]
To make sure I have formatted and labeled the RAID correctly I would like to be able to cross-reference the labels and drive paths. Currently I wasn't able to find any way to make sure that my `/dev/sda` was with a `hdd` label.

**Comment by presto8** [comment]
I agree, it would be very useful to have the info in show-super. Indeed, it is available in fs usage, but that requires the filesystem to be mounted. Having it available in show-super allows it to be visible before mounting.
